### PR TITLE
blog: windows support

### PR DESCRIPTION
**Pull request by nicks**
Hello @victorwuky, @jazzdan,

Please review the following commits I made in branch nicks/ch7044/windows:

677b541bf5b209570266efe5fd272d5131ce78f5 (2020-05-19 18:55:17 -0400)
blog: windows support

### CODE REVIEW

**Comment by nicks** [comment]
you have no idea how enthusiastic i am about this header image

File: blog/_posts/2020-05-20-windows-support.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+- Our core language examples in 
+  {[Python](https://github.com/tilt-dev/tilt-example-python), 
+   [NodeJS](https://github.com/tilt-dev/tilt-example-nodejs), 
+   [Go](https://github.com/tilt-dev/tilt-example-go), 
+   [Java](https://github.com/tilt-dev/tilt-example-java), 
+   [C#](https://github.com/tilt-dev/tilt-example-csharp)} 
+  work on Windows.
+- Every Tilt change runs our tests on Windows.
+- We maintain binaries and installers for Windows for every Tilt release.
```
Comment (victorwuky) [inline blog/_posts/2020-05-20-windows-support.md:36]:
```suggestion
- We maintain binaries and installers for Windows for [every Tilt release](https://github.com/tilt-dev/tilt/releases).
```

File: blog/_posts/2020-05-20-windows-support.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
```
Comment (victorwuky) [inline blog/_posts/2020-05-20-windows-support.md:37]:
```suggestion

In other words, Windows is now a first-class citizen alongside Linux and macOS.
```

File: blog/_posts/2020-05-20-windows-support.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  
+- [Docker for Windows](https://docs.docker.com/docker-for-windows/) seems to get
+  better every month.  We're particularly excited for the [WSL2
+  backend](https://www.docker.com/blog/new-docker-desktop-wsl2-backend/).
+
+- [Goreleaser](https://goreleaser.com/) made cross-compiling Windows binaries a
+  breeze.
+
+- [Scoop](https://scoop.sh/) is a package manager that lets you manage tools
+  without annoying permissions dialogs. Tilt's install script will use Scoop if
```
Comment (victorwuky) [inline blog/_posts/2020-05-20-windows-support.md:85]:
```suggestion
  without annoying permissions dialogs. [Tilt's install script](https://docs.tilt.dev/install.html#windows) will use Scoop if
```

File: blog/_posts/2020-05-20-windows-support.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  was a huge help - just getting all our tests to pass on Windows fixed most of
+  the bugs.
+  
+Thanks to all of these projects!
+
+## What's next?
+
+If you're a Windows user who passed on Tilt on the past, or have been using it
+on WSL, now's a great time to pick it up again!
+
```
Comment (victorwuky) [inline blog/_posts/2020-05-20-windows-support.md:99]:
```suggestion

[Let us know](https://tilt.dev/contact) how we can make Tilt better for Windows.
```

Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md:102]: done

File: blog/_posts/2020-05-20-windows-support.md (hunk 3 above)
Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md]: done

File: blog/_posts/2020-05-20-windows-support.md (hunk 2 above)
Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md:39]: done

File: blog/_posts/2020-05-20-windows-support.md (hunk 1 above)
Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md]: done

File: blog/_posts/2020-05-20-windows-support.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  - windows
+---
+
+Historically, Tilt has supported multi-service development on Linux and macOS.
+
+Some of our users hacked it to work on Windows anyway. A few pro-active
+contributors even sent us PRs occasionally to fix Windows bugs ❤. Parts of
+Tilt worked well. Other parts didn't.
+
+So we're positively chuffed to announce full Windows support!
```
Comment (jazzdan) [inline blog/_posts/2020-05-20-windows-support.md:20]: A+ use of "chuffed"

File: blog/_posts/2020-05-20-windows-support.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+   [C#](https://github.com/tilt-dev/tilt-example-csharp)} 
+  work on Windows.
+- Every Tilt change runs our tests on Windows.
+- We maintain binaries and installers for Windows for [every Tilt release](https://github.com/tilt-dev/tilt/releases).
+
+In other words, Windows is now a first-class citizen alongside Linux and macOS.
+
+Install it with:
+
+```
```
Comment (jazzdan) [inline blog/_posts/2020-05-20-windows-support.md:42]: Can we make this use powershell syntax highlighting? I'm honestly not sure.

File: blog/_posts/2020-05-20-windows-support.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+Thanks to all of these projects!
+
+## What's next?
+
+If you're a Windows user who passed on Tilt on the past, or have been using it
+on WSL, now's a great time to pick it up again!
+
+[Let us know](https://tilt.dev/contact) how we can make Tilt better for Windows.
+
+[^1]: Tilt does not (yet) support Windows-based containers.
```
Comment (jazzdan) [inline blog/_posts/2020-05-20-windows-support.md:105]: Oh interesting! If I were reading this blog post, which I am, I would be curious why

Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md:105]: added some text

File: blog/_posts/2020-05-20-windows-support.md (hunk 6 above)
Comment (nicks) [reply blog/_posts/2020-05-20-windows-support.md:42]: the answer is yes!

**Pull request merged**